settings/DevHome.Settings/ViewModels/AccountsViewModel.cs

```diff
@@ -5,14 +5,20 @@
 using System.Linq;
 using System.Threading.Tasks;
 using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
 using DevHome.Common.Contracts.Services;
 using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Settings.Models;
 using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
 
 namespace DevHome.Settings.ViewModels;
 
-public class AccountsViewModel : ObservableObject
+public partial class AccountsViewModel : ObservableObject

```

Comment:
Why partial?

Reply:
Classes the implement ObservableObject are partial, since ObservableObject generates additional code in the class.